example/src/tests/groupTests.ts

```diff
@@ -2047,12 +2037,9 @@ test('can create new installation without breaking group', async () => {
     233, 120, 198, 96, 154, 65, 132, 17, 132, 96, 250, 40, 103, 35, 125, 64,
     166, 83, 208, 224, 254, 44, 205, 227, 175, 49, 234, 129, 74, 252, 135, 145,
   ])
-  const wallet1 = new Wallet(
-    '0xc54c62dd3ad018ef94f20f0722cae33919e65270ad74f2d1794291088800f788'
-  )
-  const wallet2 = new Wallet(
-    '0x8d40c1c40473975cc6bbdc0465e70cc2e98f45f3c3474ca9b809caa9c4f53c0b'
-  )
+  const wallet1 = Wallet.createRandom()
+  const wallet2 = Wallet.createRandom()
```

Comment:
These wallets are in the bad [key package](https://github.com/xmtp/libxmtp/issues/947) state so had to use random ones. I think this is better anyways. But good to know for future testing when we have a fix.